django_mongodb_backend/fields/embedded_model.py

```diff
@@ -12,6 +12,8 @@
 class EmbeddedModelField(models.Field):
     """Field that stores a model instance."""
 
+    value_is_model_instance = True
```

Comment:
Just passing by, could this be `is_model_instance` or is it important to know that the field is considered a "value" to something else ?